platform/formats/android_xml.mdx

```diff
@@ -136,6 +137,61 @@ Since the `[` and `]` characters are not allowed in the key names in the Android
 there should be no conflicts if you use the Tolgee project only for the Android app. Otherwise, remember that keys named
 with the described syntax are converted to arrays.
 
+## Strings wrapped with CDATA
+When Tolgee imports the Android XML file, it extracts the strings from the CDATA sections. e.g.
+`<string><![CDATA[Hello]]></string>` is imported as `Hello`. Keys with imported translations with CDATA-wrapped values
+are marked with `_androidWrapWithCdata` [custom value](/platform/translation_keys/custom_values). When exporting,
+keys marked with `_androidWrapWithCdata` are wrapped with CDATA.
```

Comment:
Does the underscore needs to be there?

Reply:
Yep, all the properties which have a meaning for Tolgee are marked with the underscore. (see XLIFF, Flutter)

Reply:
😢

Reply:
Then it's ok like this I guess.